### DISCUSSION TITLE
GL-iNet Slate Plus (A1300) support & General questions

### DISCUSSION THREAD

**Discussion by elisterr** [SmoothWAN/SmoothWAN#93] [Ideas and feature requests]
Hey guys! 

First of all, thank you. You've finally done something I needed 2 years ago. Putting Speedify on a router. 
I've been using Speedify on a racetrack to broadcast and communicate with the car even when its travelling at 200kmh. Combining multiple ISP's works like a charm and doing a speedtest of 150mbits down, 100mbits up on a fast straight made me fall for Speedify and its features. 

I have been using it via the native Rapsberry Pi 4 (Rasperry OS) Speedify app + USB hub + 4 (but once even 10x) 4G modems on up to three ISP-s at once. At first learning linux and various bypasses of modeswitch to use the dongles (mostly Huawei 3372h and 3372s) but finally routing to flashing a custom firmware on the sticks to have automatic (and my desired option) modeswitch on the dongles themselves. 

I've recently bought a GL-iNet Slate Plus (A1300) to expand the WIFI capabilities and have extra LAN ports to connect to. Working fine with Tailscale package on the router BUT... 

The speed trough the A1300 is considerably slower (up to -35%) from the direct wifi connection to the Pi4.  

And now I had the fun Idea to once again Google "Speedify on OpenWRT" and found you guys! 

My two questions after a long monologue are: 

**1) Is it possible (or in planning) to have SmoothWAN on the A1300 model aswell (as the newest travel router on GL-iNet lineup)?**

**2) What are the maximum speeds achieved by SmoothWAN on a router and how many different connections can it handle?** Like said I've had up to 10 different 4G modems on a Pi before (all with externally powered USB hubs) but it seemed to not actually handle it beyond 7 or 8 modems. Meaning that the last 2 modems were just for failover as the Pi was unable to use the max throughput of those (i.e max 8-10 Mbit/s, as others managed 20-40Mbit/s).

Thank you guys! Great to see the world progress! :)

**Comment by TalalMash** [comment]
Hi,

Internet on a racetrack sounds like a fun challenge!
That's a nice setup, I'm not sure how Speedify would run on the A1300, I wouldn't expect it to go past 100Mbit since the CPU is much slower compared to the AX(T)1800 or the Netgear: 780Mhz Cortex-A7 vs 1.2-1.5Ghz Cortex-A53 not to mention acceleration and bus/RAM speeds.
I'll build one anyways along another (yet to decide) cheap globally off the shelf hardware for testing and low budget low speed setups, the Pi's shortage is unfortunate. If you have any suggestions that would be helpful!
Nano Pi is problematic with the random hardware revisions, not sure how stable is the R5S.

Regarding speed with 2 WANs that are relative, the Pi according to my tests was able to go up to ~300-350Mbit with encryption enabled, 450 Mbit is possible without encryption.
Slate AX / Flint was around 200-300Mbit, haven't got the opportunity to go higher.
Have yet to test the Netgear which should be the fast (1.8Ghz dual core Cortex-A7) 
Speedify is mostly single thread, but having more cores can help with poor kernel scheduling or network shortcuts/cut-through.

**Comment by TalalMash** [comment]
Next version will include an extensive benchmarking app that automatically tests Speedify in protocols and mode selections, it's common for ISPs to segregate and mess up standards when it comes to internet protocols.
